qcengine/programs/tests/test_ccsd.py

```diff
@@ -0,0 +1,77 @@
+import copy
+
+import pytest
+
+import qcelemental as qcel
+from qcelemental.testing import compare_values
+import qcengine as qcng
+from qcengine import testing
+
+
+_base_json = {"schema_name": "qcschema_input", "schema_version": 1}
```

Comment:
We don't need this `_base_json` stuff anymore due to Pydantic, best to remove it to prevent confusion IMO.